client/components/app-promo/index.jsx

```diff
@@ -74,20 +80,31 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
-		var element = (
+		const promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		return (
 			<div className="app-promo">
 				<span tabIndex="0" className="app-promo__dismiss" onClick={ this.dismiss } >
 					<Gridicon icon="cross" size={ 24 } />
-					<span className="screen-reader-text">{ this.translate( 'Dismiss' ) }</span>
+					<span className="app-promo__screen-reader-text screen-reader-text">
```

Comment:
I'm tempted with these sometimes too to have a separate stylesheet selector that extends `.screen-reader-text`, treating it more like a mixin (perhaps [placeholder](http://thesassway.com/intermediate/understanding-placeholder-selectors)) than a class to apply to the component directly.

Kinda outside the scope of this pull request, but a thought to consider.